docs/javascript/v2/features/notifications.mdx

```diff
@@ -131,7 +131,7 @@ const unsubscribe = hmsNotifications.onNotification((notification) => {
 
 <Tab id="notification-types-1">
 
-```jsx{14-44}
```

Comment:
what does this do?

Reply:
highlights from line-toline, originally it highlighted everything within swtich but difficult to maintain everytime